### ISSUE TITLE
[CDAI] Your focus-trap must have at least one container with at least one tabbable node in it at all times

### ISSUE THREAD

**Issue by asfordmatt** [carbon-design-system/ibm-cloud-cognitive#848]
We have been broken by v1.2.6 of CDAI components.
The break can be seen from the Storybook: IdeSlideOverPanel > With state manager

## What package(s) are you using?

- [ ] Cloud & Cognitive
- [x] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

```
Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at updateTabbableNodes (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:10558993)
    at Object.activate (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:10562913)
    at FocusTrap.componentDidUpdate (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9461187)
    at Ji (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9951122)
    at Sj (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9968775)
    at exports.unstable_runWithPriority (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9988338)
    at cg (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9912816)
    at Jj (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9965277)
    at yj (https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9961489)
    at https://ibm-cloud-cognitive.netlify.app/vendors~main.438ed8b9.iframe.bundle.js:2:9913107

```

**Comment by asfordmatt** [comment]
Probably related to the version bump of `focus-trap-react` here:
https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/831/files

**Comment by matthewgallo** [comment]
Hey @asfordmatt, I opened a [PR](https://github.com/carbon-design-system/ibm-cloud-cognitive/pull/853) that fixes this issue for you.

**Comment by matthewgallo** [comment]
Should be fixed in [v1.2.7](https://github.com/carbon-design-system/ibm-cloud-cognitive/releases/tag/%40carbon%2Fibm-cloud-cognitive-cdai%401.2.7)
